src/readbin/symbols.mli

```diff
@@ -26,3 +26,9 @@ open Bap.Std
 *)
 
 val read : ?demangle:Options.demangle -> filename:string -> arch -> mem -> string table
+
+val read_addrset : string -> Addr.Hash_set.t
+
+val write_addrset : ?filename:string -> Addr.Hash_set.t -> unit
```

Comment:
because `out_channel` abstracts the outputting target. It can capture both, standard output and a real file as an object of the same type, without this logic where some sentinel value is used as a designator of the choice made by a user to output to stdout. Moreover, user interface logic should be put as close to the user as possible, in general it is not a good idea to allow it to leak into your libraries and spread throughout the whole application